Web/3DObject/src/WebAR3DObject.xml

```diff
@@ -281,20 +281,34 @@
             </propertyGroup>
             <propertyGroup caption="Pinching">
                 <property key="mxUsePinchInteraction" type="boolean" defaultValue="false">
-                    <caption>Pinching:</caption>
+                    <caption>Scaling interaction:</caption>
                     <description />
                 </property>
                 <property key="mxPinchEnabled" type="expression" defaultValue="false">
-                    <caption>Enable pinching:</caption>
+                    <caption>Enable scaling interface:</caption>
                     <description />
                     <returnType type="Boolean" />
                 </property>
-                <property key="mxPinchToScaleEnabled" type="boolean" defaultValue="false">
-                    <caption>Enable pinch to scale:</caption>
+                <property key="mxPinchRotationEnabled" type="expression" defaultValue="false">
+                    <caption>Enable rotation interface:</caption>
```

Comment:
Yeah, it's mint be a too much, but why we use colon in caption? I though we should not use punctuation in captions.

Reply:
I just checked, and I have that in multiple places. I'll check if there is anything against it.